src/libcore/slice.rs

```diff
@@ -655,6 +655,12 @@ impl<'a, T> Default for &'a [T] {
     fn default() -> &'a [T] { &[] }
 }
 
+#[stable(feature = "mut_slice_default", since = "1.5.0")]
+impl<'a, T> Default for &'a mut [T] {
+    #[stable(feature = "mut_slice_default", since = "1.5.0")]
```

Comment:
The annotations will be respected, but this annotation (the one on the *method*) is legitmately useless. The implementation of a trait can be either stable or unstable *as a whole*. You can experimentally implement part of the trait. As such, we only need/want the annotation on the `impl` block itself.